Include/interactivity.h

```diff
@@ -1,1152 +1,1169 @@
+    /// </summary>
+    /// <param name="groupId">The unique string identifier for the group.</param>
+    interactive_group(
+        _In_ string_t groupId
+        );
+
+    /// <summary>
+    /// Constructor for the interactive_group object.
+    /// </summary>
+    /// <param name="groupId">The unique string identifier for the group.</param>
+    /// <param name="scene">The scene shown to the group.</param>
+    interactive_group(
+        _In_ string_t groupId,
+        _In_ std::shared_ptr<interactive_scene> scene
+        );
+
+    /// <summary>
+    /// Unique string identifier for the group.
+    /// </summary>
+    _MIXERIMP const string_t& group_id() const;
+
+    /// <summary>
+    /// Returns a pointer to the scene assigned to the group.
+    /// </summary>
+    _MIXERIMP std::shared_ptr<interactive_scene> scene();
+
+    /// <summary>
+    /// Assigns a scene to the group.
+    /// </summary>
+    _MIXERIMP void set_scene(std::shared_ptr<interactive_scene> currentScene);
+
+    /// <summary>
+    /// Gets all the participants assigned to the group. The list may be empty.
+    /// </summary>
+    _MIXERIMP const std::vector<std::shared_ptr<interactive_participant>> participants();
+
+private:
+
+    /// <summary>
+    /// Internal function to construct a interactive_group object.
+    /// </summary>
+    interactive_group();
+
+    std::shared_ptr<MICROSOFT_MIXER_NAMESPACE::interactivity_manager> m_interactivityManager;
+    std::shared_ptr<MICROSOFT_MIXER_NAMESPACE::interactive_group_impl> m_impl;
+
+    friend interactive_group_impl;
+    friend interactivity_manager_impl;
+    friend interactivity_manager;
+};
+
+
+/// <summary>
+/// Represents an interactive scene. These scenes are configured 
+/// using the Mixer Interactive Lab.
+/// </summary>
+class interactive_scene
+{
+public:
+
+    /// <summary>
+    /// Unique string identifier for the scene.
+    /// </summary>
+    _MIXERIMP const string_t& scene_id() const;
+
+    /// <summary>
+    /// Returns all the groups that the scene is assigned to. This list may be empty.
+    /// </summary>
+    _MIXERIMP const std::vector<string_t> groups();
+
+    /// <summary>
+    /// Returns a list of all the buttons in the scene. This list may be empty.
+    /// </summary>
+    _MIXERIMP const std::vector<std::shared_ptr<interactive_button_control>> buttons();
+
+    /// <summary>
+    /// Returns the pointer to the specified button, if it exist.
+    /// </summary>
+    /// <param name="controlId">The unique string identifier of the button.</param>
+    _MIXERIMP const std::shared_ptr<interactive_button_control> button(_In_ const string_t& controlId);
+
+    /// <summary>
+    /// Returns a list of all the joysticks in the scene. This list may be empty.
+    /// </summary>
+    _MIXERIMP const std::vector<std::shared_ptr<interactive_joystick_control>> joysticks();
+
+    /// <summary>
+    /// Returns the pointer to the specified joystick, if it exist.

```

Comment:
The commented out method didn't have one. The Unity SDK also doesn't let you get progress by participant. But yes, it seems like we should allow this. I'll look into it, but probably for another PR. I feel like there was a reason we couldn't do it by participant, but could be misremembering.